src/workflows/integration/totalvi_leiden/config.vsh.yaml

```diff
@@ -16,57 +16,34 @@ argument_groups:
         description: ID of the sample.
         example: foo
       - name: "--input"
-        required: true
+        alternatives: ["-i"]
         type: file
-        description: Path to the sample.
-        example: dataset.h5mu
-      - name: "--layer"
-        type: string
-        description: use specified layer for expression values instead of the .X object from the modality.
-        required: false
-      - name: "--modality"
-        description: Which modality to process.
+        description: Input h5mu file to be integrated.
+        example: input.h5mu
+        direction: input
+        required: true
+      - name: "--modality_rna"
         type: string
         default: "rna"
         required: false
-      - name: "--prot_modality"
-        description: Which modality to process.
+      - name: "--modality_protein"
```

Comment:
Could you check what is the most used naming for the modality arguments in order to align? 
In the `totalvi_scarches_leiden` workflow, `prot_modality` is being used. 
This comment will probably also be applicable to other components in this PR